src/server/handlers.rs

```diff
@@ -54,22 +54,15 @@ impl<P: Publisher> Handler<P> {
             "Manifest is valid. Fetching current manifests from store"
         );
 
-        if manifest.version() == LATEST_VERSION {
-            self.send_error(
-                msg.reply,
-                format!("A manifest with a version {LATEST_VERSION} is not allowed in wadm"),
-            )
-            .await;
+        let manifest_validation_output = validate_manifest_version(manifest.version());
+        if let Some(first_err) = manifest_validation_output.errors().first() {
+            self.send_error(msg.reply, format!("invalid manifest version: {first_err}"))
```

Comment:
I think we should probably return a comma delimited list of all errors so the user doesn't have to fix one, then find out about the next one

Reply:
Sure! I don't think the `send_error` machinery is really built for this (and this really isn't the right place to be catching failures anyway) but made this change!